### PR TITLE
feat(design-v2): Call control buttons

### PR DESCRIPTION
**Pull request by kristian-mkd**
## 🎯 Goal
- Jira Ticket: [PBE-5857](https://stream-io.atlassian.net/browse/PBE-5857) 
- Add new call control buttons

## 🛠 Implementation details

✅ ParticipantInfo button is moved to the bottom call controls sections
✅ Around half of the call control buttons in the bottom section are moved to bottom drawer and the remaining main buttons are split in left and right sections
✅ Add more button is added that will open bottom drawer with more call controls
✅ New styling and icons is applied on the call controls

## 🎨 UI Changes

<summary>iOS</summary>
<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
ios-after
            </td>
            <td>
     ios-after
</td>
        </tr>
    </tbody>
</table>

<details>
<summary>Android</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
     ios-after
            </td>
            <td>
     ios-after
            </td>
        </tr>
    </tbody>
</table>
</details>

[PBE-5857]: https://stream-io.atlassian.net/browse/PBE-5857?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/react-native-sdk/src/components/Call/CallContent/CallContent.tsx [hunk 1]
```diff
@@ -201,67 +197,82 @@ export const CallContent = ({
...
+        mainContainer: { flex: 1 },
+        callContent: { flex: 1 },
+        safeArea: { flex: 1 },
+        unsafeArea: {
+          position: 'absolute',
+          left: 0,
+          right: 0,
+          bottom: 0,
+          // TODO: should be replaced with safeAreaInsets.bottom for all devices on ios and android
+          height: 34,
```
Comment (kristian-mkd) [inline packages/react-native-sdk/src/components/Call/CallContent/CallContent.tsx:268]: @santhoshvai what do you think about using the `react-native-safe-area-context` in the SDK? We use it in the sample apps, but I think we need it in the sdk as well.

Comment (santhoshvai) [reply packages/react-native-sdk/src/components/Call/CallContent/CallContent.tsx:268]: strict NO here

the components can go inside bottom tabs, inside headers and what not

we should not freely assume that its always inside a whole screen

Comment (santhoshvai) [reply packages/react-native-sdk/src/components/Call/CallContent/CallContent.tsx:268]: ideally if wrapped under safe area the padding is applied somehow

for example, padding is passed through theme or props

**Review by oliverlaz — APPROVED** [review]

File: packages/react-native-sdk/src/components/Call/CallContent/CallContent.tsx (hunk 1 above)
Comment (kristian-mkd) [reply packages/react-native-sdk/src/components/Call/CallContent/CallContent.tsx:268]: Great points! It makes complete sense, I will try to add it to theme or passable by props by integrator 👍

Comment (kristian-mkd) [reply packages/react-native-sdk/src/components/Call/CallContent/CallContent.tsx]: I added the insets in the theme with this [commit](https://github.com/GetStream/stream-video-js/pull/1514/commits/2495166a054ae4f91aaf1a87eae792c911349e94). If you approve the solution I will create a cookbook for this.

File: packages/react-native-sdk/src/components/Call/CallControls/BadgeCountIndicator.tsx
Comment (santhoshvai) [inline packages/react-native-sdk/src/components/Call/CallControls/BadgeCountIndicator.tsx:1]: do we need this badge inside the SDK?

Comment (kristian-mkd) [reply packages/react-native-sdk/src/components/Call/CallControls/BadgeCountIndicator.tsx:1]: We don't, moved to dogfood app with this commit https://github.com/GetStream/stream-video-js/pull/1514/commits/4e2f9ba174dc8f0261904adb38a38de4356884c0

File: packages/react-native-sdk/src/components/Call/CallContent/CallContent.tsx [hunk 2]
```diff
@@ -201,67 +197,82 @@ export const CallContent = ({
...
       {!disablePictureInPicture && (
         <RTCViewPipIOS
           includeLocalParticipantVideo={iOSPiPIncludeLocalParticipantVideo}
         />
       )}
-      <View style={[styles.container, landscapeStyles, callContent.container]}>
-        <View style={[styles.container, callContent.callParticipantsContainer]}>
+      <View style={styles.mainContainer}>
+        <View style={styles.unsafeArea} />
+        <SafeAreaView style={[styles.safeArea, callContent.container]}>
```
Comment (santhoshvai) [inline packages/react-native-sdk/src/components/Call/CallContent/CallContent.tsx:209]:
```suggestion
        <SafeAreaView style={[styles.safeArea, callContent.container]}>
```
Please do not use SafeAreaView inside the SDK

Comment (santhoshvai) [reply packages/react-native-sdk/src/components/Call/CallContent/CallContent.tsx:209]: also check and remove anywhere else where you use it, insets from theme should be applied as padding

**Review by santhoshvai — APPROVED** [review]

**Pull request merged**